openedx/core/djangoapps/notifications/models.py

```diff
@@ -109,6 +109,7 @@ class Notification(TimeStampedModel):
     email = models.BooleanField(default=False, null=False, blank=False)
     last_read = models.DateTimeField(null=True, blank=True)
     last_seen = models.DateTimeField(null=True, blank=True)
+    group_by_id = models.CharField(max_length=255, null=True, default="")
```

Comment:
I think that charfield 225 is too much for id,  and max_length  20 would be sufficient.

Reply:
32 characters I think to be safe

Reply:
We will be running queries on this field so might be a good idea to index it

Reply:
I am planning to set it to 36 characters (equal to length of UUID)